forest/src/daemon.rs

```diff
@@ -37,6 +37,12 @@ pub(super) async fn start(config: Config) {
         .set(address::Network::Mainnet)
         .unwrap();
 
+    // Configure the Rayon Threadpool
+    rayon::ThreadPoolBuilder::new()
+        .thread_name(|n| format!("rayon-thread-{}", n))
+        .build_global()
+        .unwrap();
```

Comment:
Bringing this up in anticipation of our style guide: When do you prefer `unwrap` over `expect`?

Reply:
Good point. I think expect is much more appropriate here